pytest_fixtures/core/contenthosts.py

```diff
@@ -232,6 +232,22 @@ def module_container_contenthost(request, module_target_sat, module_org, module_
         yield host
 
 
+@pytest.fixture(scope='module')
+def module_flatpak_contenthost(request):
+    request.param = {
+        "rhel_version": "9",
+        "distro": "rhel",
+        "no_containers": True,
+    }
+    with Broker(**host_conf(request), host_class=ContentHost) as host:
+        host.register_to_cdn()
+        for pkg in ['podman', 'flatpak', 'dbus-x11']:
+            res = host.execute(f'dnf -y install {pkg}')
+            assert res.status == 0, f'{pkg} installation failed: {res.stderr}'
```

Comment:
Can this be a single DNF transaction?
```suggestion
        pkgs = ['podman', 'flatpak', 'dbus-x11']
        res = host.execute(f'dnf -y install {' '.join(pkgs)}')
        assert res.status == 0, f'Package installation failed: {res.stderr}'
```

Reply:
@ogajduse It could be. I just wanted to keep it separate to provide more info which installation failed and why through `res.stderr`. But it doesn't matter so much probably, dnf should return some meaningful info anyway.